kernel/base/kernel.json

```diff
@@ -1,5 +1,5 @@
 {
-  "argv": [ "java", "-jar", "__PATH__", "{connection_file}" ],
```

Comment:
i think this file should be deleted since the base kernel never runs.

Reply:
done.